File: C++/Inheritance/single_inheritance.cpp
//WAP of single inheritance..
#include<iostream>
using namespace std;

class Base{
    private :
     int a,b;
    public :
        void get(){
            cout<<"Enter the numbers :";
            cin>>a>>b;
        }
        void show(){
            cout<<a<<"\t";
            cout<<b<<"\n";
        }

};
class Derive:private Base{
    private :
    public :
    void display(){
        get();
        show();
        cout<<"Call of base function..";
    }
};
int main(){
    system("clear");
    Derive d;
    //d.get();
    //d.show();
    d.display();
}

File: Data_Structure_and_Algorithm/src/Inheritance/Single_Inheritance.java
package Inheritance;

class Base{
	int a=10;
	void show() {
		System.out.println(a);
	}
}
class Derived extends Base{
	int b=20;
	void display() {
		System.out.println(b);
	}
}
public class Single_Inheritance {
	public static void main(String arg[]) {
		
		Derived D=new Derived();
		D.show();

		D.display();
	}
}
